### PR TITLE
feat: add product listing page

### PR DESCRIPTION
**Pull request by ravikumar0403**
Change log:
Added product listing page
image

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .hero{
-    background-image: linear-gradient(0deg, rgba(0, 0, 0, 0.7), rgba(0, 0, 0, 0.7)), url(./assets/hero.jpg);
+    background-image: linear-gradient(0deg, rgba(0, 0, 0, 0.7), rgba(0, 0, 0, 0.7)), url(../assets/hero.jpg);
```
Comment (sonishreyas) [inline css/style.css:2]: Just a suggestion, you can use GitHub raw URL here so it won't have a dependency on folder structure

File: pages/products.html [hunk 2]
```diff
@@ -0,0 +1,255 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (sonishreyas) [inline pages/products.html:7]: an author tag can be added here

**Review by sonishreyas — APPROVED** [review]
I have kept a few suggestions rest looks good to me

**Comment by parthpandyappp** [comment]
Review strongly aligns with @sonishreyas . Rest LGTM :+1:

**Review by parthpandyappp — COMMENTED** [review]
Good work though, the extent of modularity in code is fantastic :+1:

**Pull request merged**